src/main/java/com/develhope/spring/deals/controllers/RentalController.java

```diff
@@ -20,4 +21,11 @@ public class RentalController {
     public RentalReturnerDTO create(@RequestBody RentalCreatorDTO rentalCreatorDTO) {
         return rentalService.create(rentalCreatorDTO);
     }
+
+    @DeleteMapping("/{rentalId}")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
No content si rifa ad un errore dove non viene trovato il contenuto, in questo caso se l'azione va a buon fine passerei un Ok o Accepted:)